test/features/bootstrap_bugs.feature

```diff
@@ -64,3 +64,18 @@ Feature: Regression test for bootstrap bugs
     And     Service "hawk.service" is "stopped" on "hanode2"
     When    Run "crm corosync get nodelist.node.ring0_addr" on "hanode1"
     Then    Expected "10.10.10.3" not in stdout
+
+  @clean
+  Scenario: Multi nodes join in parallel(bsc#1175976)
```

Comment:
This looks great!!!
Maybe we would need an additional test to check the `joining` nodes wait if the `init` node is still in progress.
This way, we would cover all of the major functionalities that we were looking for with this lock mechanism

Reply:
For this scenario: ` Multi nodes join in parallel(bsc#1175976)`
Maybe we could find a way to let the join process insist/sleep over 120s, then other join node will exit
I'm not sure how to make this happen without changing source code...